GiniVision/Classes/Core/CameraViewController.swift

```diff
@@ -14,20 +14,13 @@ import AVFoundation
      Called when a user takes a picture, imports a PDF/QRCode or imports one or several images.
      Once the method has been implemented, it is necessary to check if the number of
      documents accumulated doesn't exceed the minimun (`GiniImageDocument.maxPagesCount`).
-     Afterwards, it is mandatory to call the `DocumentValidationHandler` block passing the error
-     `CameraError.maxFilesPickedCountExceeded` (if the page count limit was exceeded) and
-     the inner completion block that will be executed once the gallery dismissal animation completes.
      
      - parameter viewController: `CameraViewController` where the documents were taken.
      - parameter documents: One or several documents either captured or imported in
      the `CameraViewController`. They can contain an error produced in the validation process.
-     - parameter completion: `DocumentPickerCompletion` block used to check if there is an issue with
-     the captured documents. The completion block also has an inner block that is executed once the
-     picker has been dismissed when there are no errors.
      */
     @objc func camera(_ viewController: CameraViewController,
-                      didCaptureDocuments documents: [GiniVisionDocument],
-                      validationHandler: DocumentValidationHandler?)
+                      didCapture document: GiniVisionDocument)
```

Comment:
You changed the public API meaning clients will have to do changes even without using multi-page. Can you instead deprecate old methods and still use them and make the new methods mandatory for multi-page?